Integrations/integration-ICEBRG.yml

```diff
@@ -0,0 +1,278 @@
+commonfields:
+  id: ICEBRG
+  version: -1
+name: ICEBRG
+display: ICEBRG
```

Comment:
we don't have all capital display names, names, etc.
its usually non capital in id and name, and first letter in display